psi4/driver/procrouting/proc.py

```diff
@@ -873,7 +873,11 @@ def select_ccsd_t__gradient(name, **kwargs):
     # Considering only ccenergy
 
     func = None
-    if reference in ['RHF', 'UHF']:
+    if reference in ['RHF']:
+        if mtd_type == 'DF':
+            if module in ['', 'OCC']:
+                func = run_dfocc_gradient
+    elif reference == 'UHF':
```

Comment:
This change loses the ability to run RHF CONV gradients via ccenergy. Easy to edit it back.

Reply:
Actually, we already patched that up once [here](https://github.com/loriab/privUgurPsi4/commit/d4bbc369fa8a34871bd3e8950ffc4267354525d8). Maybe see if any other improvements on that branch didn't make it to this PR?